src/v2/view-builder/views/password/ReEnrollAuthenticatorPasswordView.js

```diff
@@ -17,7 +17,8 @@ const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
   },
 
   getPasswordPolicySettings () {
-    return this.options.appState.get('recoveryAuthenticator').settings;
+    return this.options.appState.get('recoveryAuthenticator')?.settings
+      || this.options.appState.get('enrollmentAuthenticator')?.settings;
```

Comment:
For more info, please refer to @SamSanjabi-okta 's comment here - https://oktainc.atlassian.net/browse/OKTA-349016?focusedCommentId=1641471